ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerFeatureLayer/LocalServerFeatureLayer.cpp

```diff
@@ -81,9 +80,15 @@ void LocalServerFeatureLayer::connectSignals()
     {
       qDebug() << "Local server started";
 
-      // set temp path
-      QTemporaryDir tempDir;
-      LocalServer::instance()->setTempDataPath(tempDir.path());
+      // create temp path
+      const QString tempPath = QDir::homePath() + "/EsriQtTemp";
+
+      // create the directory
+      if (!QDir(tempPath).exists())
+        QDir().mkdir(tempPath);
```

Comment:
This applies to all. Rather than creating two QDir temporaries here, would this work?

```c++
QDir dir(tempPath);
if (dir.exists())
  dir.mkdir(tempPath);
```
```